src/main/java/net/atos/zac/policy/PolicyService.java

```diff
@@ -82,11 +82,15 @@ public ZaakRechten readZaakRechten(final Zaak zaak) {
     }
 
     public ZaakRechten readZaakRechten(final Zaak zaak, final ZaakType zaaktype) {
+        return readZaakRechten(zaak, zaaktype, null);
+    }
+
+    public ZaakRechten readZaakRechten(final Zaak zaak, final ZaakType zaaktype, final LoggedInUser user) {
         final ZaakData zaakData = new ZaakData();
         zaakData.open = zaak.isOpen();
         zaakData.zaaktype = zaaktype.getOmschrijving();
-        return evaluationClient.readZaakRechten(new RuleQuery<>(new ZaakInput(loggedInUserInstance.get(), zaakData)))
-                .getResult();
+        LoggedInUser loggedInUser = user == null ? loggedInUserInstance.get() : user;
```

Comment:
Would be much cleaner in Kotlin.. ;-) If you need to make the `user` parameter nullable (is that really needed? I guess so) than for now maybe add `@Nullable` annotation in front of it just so that it is clear to everyone. In Kotlin later we can then make it nullable too. Just a minor thing. Null handling if so messy in Java.. ;-)